shared/common-adapters/user-proofs.desktop.js

```diff
@@ -10,14 +10,16 @@ import {defaultColor} from '../common-adapters/icon.shared'
 import {globalStyles, globalColors, globalMargins} from '../styles'
 import {metaNone, checking as proofChecking} from '../constants/tracker'
 
-function MissingProofRow ({missingProof, style}: {missingProof: MissingProof, style: Object}): React$Element<*> {
+function MissingProofRow ({missingProof}: {missingProof: MissingProof}): React$Element<*> {
   const missingColor = globalColors.black_20
   return (
-    <Box style={{...styleMissingProofRow, ...style}} className='user-proof-row' key={missingProof.type} onClick={() => missingProof.onClick(missingProof)}>
-      <Icon className='user-proof-row__icon' style={{...styleService, color: missingColor}} type={shared.iconNameForProof(missingProof)} hint={missingProof.type} />
+    <Box style={styleMissingProofRow} className='user-proof-row' key={missingProof.type} onClick={() => missingProof.onClick(missingProof)}>
+      <Box style={iconContainer}>
```

Comment:
we center the icon in a fixed size box which can be on top so the alignment is always perfect